src/main/java/by/itech/lab/supplier/service/impl/PriceCalculationServiceImpl.java

```diff
@@ -0,0 +1,76 @@
+package by.itech.lab.supplier.service.impl;
+
+import by.itech.lab.supplier.dto.AddressDto;
+import by.itech.lab.supplier.dto.ApplicationDto;
+import by.itech.lab.supplier.dto.ApplicationItemDto;
+import by.itech.lab.supplier.dto.WarehouseItemDto;
+import by.itech.lab.supplier.service.PriceCalculationService;
+import by.itech.lab.supplier.service.TaxService;
+import by.itech.lab.supplier.service.WarehouseService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class PriceCalculationServiceImpl implements PriceCalculationService {
+
+    private final static Integer EARTH_RADIUS = 6371000; //meters
```

Comment:
We can put the note about meters into the name of the variable to avoid using comment.